cron/calculateAssessmentMode.sql

```diff
@@ -0,0 +1,39 @@
+WITH assessment_instance_modes AS (
+    SELECT
+        count(ai.id) as count,
+        ai.mode,
+        a.id AS assessment_id
+    FROM
+        assessment_instances AS ai
+        JOIN assessments AS a ON (ai.assessment_id = a.id)
+    GROUP BY
+        ai.mode,
+        a.id
+),
+assessment_modes AS (
```

Comment:
Doesn't seem to work for me.
Can we have a group-by without an aggregate function? And I thought the order-by is executed after the group-by.